Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/package.json

```diff
@@ -39,7 +41,6 @@
     "cypress-plugin-api": "^2.10.3",
     "eslint": "^8.32.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "npm-install-peers": "^1.2.2",
-    "typescript": "^4.5.5"
```

Comment:
Are we planning on fully removing typescript from the tests? Ideally we would switch them all over to typescript at some point